src/modules/commander/HealthAndArmingChecks/checks/estimatorCheck.cpp

```diff
@@ -670,6 +670,29 @@ void EstimatorChecks::checkGps(const Context &context, Report &reporter, const s
 			mavlink_log_critical(reporter.mavlink_log_pub(), "GPS reports critical jamming state\t");
 		}
 	}
+
+	if (vehicle_gps_position.spoofing_state == sensor_gps_s::SPOOFING_STATE_INDICATED) {
+		/* EVENT
+		 */
+		reporter.armingCheckFailure(NavModes::None, health_component_t::gps,
+					    events::ID("check_estimator_gps_spoofing_indicated"),
```

Comment:
We should get all of this consolidated under GNSS checks rather than estimator check. I'll propose something in a follow up.